WordPress/src/main/java/org/wordpress/android/designsystem/DesignSystemComponentsScreen.kt

```diff
@@ -0,0 +1,52 @@
+package org.wordpress.android.designsystem
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.tooling.preview.Preview
+import org.wordpress.android.R
+
+@Composable
+fun DesignSystemComponentsScreen(
+    modifier: Modifier = Modifier
+){
+    Column(
```

Comment:
Have you considered [using a LazyList](https://youtu.be/1ANt65eoNhQ) here instead (the number of items are a few, still prefer using it)

Reply:
I didn't think it was necessary since there's only a few items expected, but I agree that it's cleaner to use 👍  Updated